### PR TITLE
[12.0][MIG] stock_orderpoint_manual_procurement_uom

### PR DESCRIPTION
**Pull request by kittiu**
Migrated to v12

Dependencies
- [x] stock_orderpoint_uom https://github.com/OCA/stock-logistics-warehouse/pull/568
- [x] stock_orderpoint_manual_procurement https://github.com/OCA/stock-logistics-warehouse/pull/557

### CODE REVIEW

**Comment by kittiu** [comment]
Wait for other dependencies to be merged.

**Comment by rousseldenis** [comment]
> Wait for other dependencies to be merged.

@kittiu Usually, it's advised to put the link to dependencies (with a checkbox) in the top of the PR to help reviewers going there first.

**Comment by LoisRForgeFlow** [comment]
This one is ready to be rebased, all dependencies are merged.

**Comment by kittiu** [comment]
Added more test to pass test check

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1: 

Thanks

File: stock_orderpoint_manual_procurement_uom/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "version": "12.0.1.0.0",
+    "author": "Eficent, "
+              "Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "category": "Warehouse Management",
+    "depends": [
+        "stock_orderpoint_uom",
+        "stock_orderpoint_manual_procurement",
+    ],
+    "data": [],
```
Comment (tbaden) [inline stock_orderpoint_manual_procurement_uom/__manifest__.py]: if you dont use the tag, just delete it

**Review by tbaden — CHANGES_REQUESTED** [review]
thanks for the work.
I did a code review. just a minor detail.

File: stock_orderpoint_manual_procurement_uom/__manifest__.py (hunk 1 above)
Comment (kittiu) [reply stock_orderpoint_manual_procurement_uom/__manifest__.py]: Fixed. Thank you!

**Review by tbaden — APPROVED** [review]

**Review by rousseldenis — APPROVED** [review]

**Pull request merged**